tests/sentry/test_prevent_file_modifications.py

```diff
@@ -0,0 +1,30 @@
+from hashlib import md5
```

Comment:
I'd rather name this file specific to the file we're locking -- rather than encourage this as a common practice (which is what it looks like given the current file naming)